app/src/main/java/com/example/criminalintentapp/MainActivity.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.criminalintentapp
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        val currentFragment = supportFragmentManager.findFragmentById(R.id.fragment_container)
+
+        if (currentFragment == null) {
+            val fragment = CrimeFragment()
```

Comment:
Usually, the newInstance pattern is used when creating fragments, to pass arguments.
Since this one does not have arguments, you can leave it as is if you want to, but I find it always odd to see fragment constructors being called like this.
See https://medium.com/@azjkjensen/using-the-newinstance-pattern-in-kotlin-e40c1b4ba1ef